### PR TITLE
feat(JAQPOT-186): Update jaqpotpy-inference

### PR DESCRIPTION
**Pull request by vassilismin**
With this commit we can take predictions from models of SklearnModel class.

### CODE REVIEW

File: src/handlers/predict.py [hunk 1]
```diff
@@ -1,32 +1,19 @@
...
-    if model.doa:
-        results['AD'] = model.doa.IN
-    else:
-        results['AD'] = [None for _ in range(len(model.prediction))]
-
-    if model.probability:
-        results['Probabilities'] = [list(prob) for prob in model.probability]
-    else:
-        results['Probabilities'] = [[] for _ in range(len(model.prediction))]
+    results = {model.dependentFeatures[0]['name']: [str(item) for item in prediction]}
```
Comment (alarv) [inline src/handlers/predict.py:13]: This should be
```suggestion
    results = {model.dependentFeatures[0]['key']: [str(item) for item in prediction]}
```

Also what happens if the model has more than 1 labels/predictedFeatures?

**Review by alarv — COMMENTED** [review]
Left 1 comment, but other than that looks good!

File: src/handlers/predict.py [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
+        probabilities = [None for _ in range(len(prediction))]
 
-    if model.doa:
-        results['AD'] = model.doa.IN
+    results = {}
+    if len(model.dependentFeatures) == 1:
+        key = model.dependentFeatures[0]['key']
+        values = [str(item) for item in prediction]
+        results[key] = values
     else:
```
Comment (alarv) [inline src/handlers/predict.py:18]: No need for the first part of the if, the else part is also covering the first case, right?

Comment (vassilismin) [reply src/handlers/predict.py:18]: No, prediction has different shape in each case

**Review by alarv — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**